cmd/admin_user_delete.go

```diff
@@ -49,7 +50,7 @@ func runDeleteUser(c *cli.Context) error {
 	if err := initDB(ctx); err != nil {
 		return err
 	}
-
+	audit.Init()
```

Comment:
Hmm .... I also think this could be a problem. The `audit.Init()` is called at many places, I can't figure out clearly when and why it is needed, or why somewhere else it is not needed.

Reply:
`audit.Init()` is only called in `/cmd`. We don't currently have a single `Init` method for commands but we should have one because you can't really be sure what side effects a command has and what modules should be inited before using them.